README.md

```diff
@@ -6,36 +6,149 @@
-this unit. We first want students to understand that `this.props.children` just
-refers to any components passed in as children.
+In React, a component can have one, many or no children. Consider the following code:
 
-A good way to demonstrate this is with a page layout component, e.g.,
+```js
+<VideoPlayer>
+  <VideoHeader>
+    <h1 className="video-title">The Simpsons</h1>
+  </VideoHeader>
+  <VideoControls />
+</VideoPlayer>
+```
+
+In this example, the `VideoPlayer` has two children: `VideoHeader` and `VideoControls`. `VideoHeader`, in turn, has one
+child: the `h1` with the title content. `VideoControls`, on the other hand, has no children.
+
+Why is this important? As you can see above, we can use children to compose our interface. For a more concrete example,
+let's say we're creating a `<Panel>` component that allows us to add content to it. Using a panel might look a little
+like this:
 
+```js
+<Panel title="Browse for movies">
+  <div>Movie stuff...</div>
+  <div>Movie stuff...</div>
+  <div>Movie stuff...</div>
+  <div>Movie stuff...</div>
+</Panel>
 ```
-class Layout extends Component {
+
+As you can see, we're adding content *inside* of the `<Panel>` tags. Now, how do we render that content in our
+component? We access it through **`this.props.children`** — a special prop that is passed to components automatically.
+
+```js
+export default class Panel extends React.Component {
   render() {
     return (
-      <div>
-        <header style={{ ... }}>Header; maybe some links</header>
-        {this.props.children}
-        <footer style={{ ... }}>Footer</footer>
+      <div className="panel">
+        <div className="panel-header">{this.props.title}</div>
+        <div className="panel-body">{this.props.children}</div>
       </div>
-    )
+    );
   }
 }
 ```
 
-Have students practice writing and rendering a component like this with
-different child components. Make sure to have at least one example that uses
-an array of siblings as `this.props.children`.
+If something like `this.props.children` didn't exist, we'd have to pass in all of our content through a prop, which
+would be very unwieldy and look really ugly:
+
+```js
+<Panel title="Browse for movies" body={<div><div>Movie stuff...</div>
+                                              <div>Movie stuff...</div>
+                                              <div>Movie stuff...</div>
+                                              <div>Movie stuff...</div></div>} />
+```
 
-Then start walking through the `React.Children` API. These methods should be
-easy to understand once students have a grip on how `this.props.children` works.
+_And_ we'd have to wrap it in an enclosing `div`! Thankfully, we can just nest it inside of the component like we did
+above, much like we nest regular HTML elements.
 
+## React.Children
+Since `this.props.children` can have one element, multiple elements or none at all, its value is respectively
```

Comment:
"Since `this.props.children` can have one element, multiple elements or none at all..." -> "Since `this.props.children` can have one element, multiple elements**,** or none at all..." (missing comma)